pkg/storaged/stratis-details.jsx

```diff
@@ -781,6 +888,32 @@ export function start_pool(client, uuid, show_devs) {
                         unlock_with_keydesc(key_desc);
                 }));
     }
+
+    if (stopped_props.key_description &&
+        stopped_props.key_description.t == "(bv)" &&
+        stopped_props.key_description.v[0]) {
+        if (stopped_props.key_description.v[1].t != "(bs)" ||
+            !stopped_props.key_description.v[1].v[0]) {
+            dialog_open({
+                Title: _("Error"),
+                Body: _("This pool can not be unlocked here because its key description is not in the expected format.")
+            });
+            return;
+        }
+        key_desc = stopped_props.key_description.v[1].v[1];
+    }
+
+    if (!key_desc) {
+        // Not an encrypted pool, just start it
+        return start();
+    } else {
+        if (stopped_props.clevis_info &&
+            stopped_props.clevis_info.t == "(bv)" &&
+            stopped_props.clevis_info.v[0]) {
+            return start("clevis").catch(unlock_with_keyring);
+        } else
+            return unlock_with_keyring();
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230522-090630-61e0720a-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)